index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
It was my first idea but I was worried to break the retro-compatibility for other users.
If for you there's no problem I can proceed